recipe/meta.yaml

```diff
@@ -60,6 +60,8 @@ outputs:
         - cython                                 # [build_platform != target_platform]
         - numpy                                  # [build_platform != target_platform]
         - gnuconfig                              # [osx and arm64]
+        # Add abseil to try to get static linked version aligned across grpc-cpp in build and host
+        - abseil-cpp
```

Comment:
Need to bump the version in the `meta.yaml` here in hindsight before merging due to this change